ci/check-params-env.sh

```diff
@@ -91,143 +91,178 @@ function check_image_variable_matches_name_and_commitref() {
     local expected_build_name  # Why some of the images has `-amd64` suffix and others not?
     case "${image_variable}" in
         odh-minimal-notebook-image-n)
+            expected_name="odh-notebook-jupyter-minimal-ubi9-python-3.11"
+            expected_commitref="release-2024b"
+            expected_build_name="jupyter-minimal-ubi9-python-3.9-amd64"
```

Comment:
these aren't correct, similar to other occurrences - though, the CI failed so it's clear and should be fixed in the followup then